test/integration/components/RecapComponent.js

```diff
@@ -0,0 +1,8 @@
+firstPersonBirthdate: 'individu-block:nth-of-type(1) strong',
+secondPersonBirthdate: 'individu-block:nth-of-type(2) strong',
+thirdPersonBirthdate: 'individu-block:nth-of-type(3) strong',
+fourthPersonBirthdate: 'individu-block:nth-of-type(4) strong',
+housingStatus: '#logement-block b',
+salaryTabLink: '.recap-revenusActivite a',
+salaryFirstMonth: '.recap-revenusActivite .row :nth-child(2)',
+salaryLast12Months: '.recap-revenusActivite hr + .row :nth-child(2)',
```

Comment:
Not sure I understand. `.recap-revenusActivite hr + .row:nth-child(2)` would have a different behaviour. Can you explicit the change you recommend ?